docs/rules/spot/parameters-not-in-body.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: /docs/cli/rules/spot/parameters-not-in-body
+---
+
+# parameters-not-in-body
+
+Requires the `in` section inside `parameters` must not contain a `body`.
+
+| ARAZZO | Compatibility |
+| ------ | ------------- |
+| 1.0.0  | ✅            |
+
+```mermaid
+flowchart TD
+
+Root ==> workflows --> workflow --> parameters
+
+style Contact fill:#codaf9,stroke:#0044d4,stroke-width:5px
+```
+
+```mermaid
+flowchart TD
+
+Root ==> x-parameters
+
+style Contact fill:#codaf9,stroke:#0044d4,stroke-width:5px
+```
+
+```mermaid
+flowchart TD
+
+Root ==> workflows --> workflow --> steps --> step --> parameters
+
+style Contact fill:#codaf9,stroke:#0044d4,stroke-width:5px
+```
+
+## API design principles
+
+This is `Spot` specific rule.
```

Comment:
We might want to label it as a separate spot version compatibility rather than Arazzo compatibility. Is the current version of spot a `v2` ?

Reply:
Current Spot versions going with v 0.7.x.